src/components/IncreamentDecreament.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import InputSpinner from "react-native-input-spinner";
```

Comment:
Do it as `import Spinner from 'react-native-input-spinner';` (or if this aliasing won't work, then `import { InputSpinner as Spinner } from 'react-native-input-spinner';`)